features/animal.feature

```diff
@@ -1 +1,21 @@
-#Create your feature here
+Feature: Animal
+  To ensure the client can track their animals
+  By name, age, type, and if they are older than a certain age
+
+  Scenario Outline: check attributes of different animals
+    Given a <type> named <name> who is <age> years old
+    Then the name should be <name>
+    And the age should be <age>
+    And the type should be <type>
+
+  Examples:
+    | name | age | type |
+    | Lucy |   4 |  dog |
+    | Carl |   2 |  cat |
+    |  Bob |   1 | deer |
+
+  Scenario: check if an animal is old
```

Comment:
I would personally name this scenario "Animal is old", though I'm not sure if either is more aligned with "the Cucumber way".